File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// CatherineShulmansQuotes SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class CatherineShulmansQuotesMakeContext
{
    public static function call(array $ctxmap, ?CatherineShulmansQuotesContext $basectx): CatherineShulmansQuotesContext
    {
        return new CatherineShulmansQuotesContext($ctxmap, $basectx);
    }
}

File: py/catherineshulmansquotes_sdk/utility/make_context.py
# CatherineShulmansQuotes SDK utility: make_context

from catherineshulmansquotes_sdk.core.context import CatherineShulmansQuotesContext


def make_context_util(ctxmap, basectx):
    return CatherineShulmansQuotesContext(ctxmap, basectx)
